files.go

```diff
@@ -72,19 +72,6 @@ func newFilesClusterCommon(cluster Cluster) Files {
 			AbsolutePath: "/etc/kubernetes/ssl/etcd/client-key.pem",
 			Data:         cluster.EtcdServer.Key,
 		},
-		// Calico Etcd client.
-		{
-			AbsolutePath: "/etc/kubernetes/ssl/etcd/calico-client-ca.pem",
-			Data:         cluster.CalicoEtcdClient.CA,
-		},
-		{
-			AbsolutePath: "/etc/kubernetes/ssl/etcd/calico-client-crt.pem",
-			Data:         cluster.CalicoEtcdClient.Crt,
-		},
-		{
-			AbsolutePath: "/etc/kubernetes/ssl/etcd/calico-client-key.pem",
-			Data:         cluster.CalicoEtcdClient.Key,
```

Comment:
Can we keep `CalicoEtcdClient` and remove `CalicoClient` instead? I think it's clearer to have `Etcd` in the name.